AdminUi/src/AdminUi/ClientApp/src/app/services/quotas-service/quotas.service.ts

```diff
@@ -5,45 +5,52 @@ import { HttpResponseEnvelope } from 'src/app/utils/http-response-envelope';
 import { environment } from 'src/environments/environment';
 
 @Injectable({
-    providedIn: 'root',
+  providedIn: 'root',
 })
 export class QuotasService {
-    apiUrl: string;
-
-    constructor(private http: HttpClient) {
-        this.apiUrl = environment.apiUrl;
-    }
-
-    getMetrics(): Observable<HttpResponseEnvelope<Metric>> {
-        return this.http.get<HttpResponseEnvelope<Metric>>(
-            this.apiUrl + '/Metrics'
-        );
-    }
-
-    getPeriods(): string[] {
-        return ['Hour', 'Day', 'Week', 'Month', 'Year'];
-    }
-
-    createTierQuota(
-        quota: Quota,
-        tierId: string
-    ): Observable<HttpResponseEnvelope<Quota>> {
-        return this.http.post<HttpResponseEnvelope<Quota>>(
-            this.apiUrl + '/Tiers/' + tierId + '/Quotas',
-            quota
-        );
-    }
-}
+  apiUrl: string;
 
-export interface Quota {
-    tierId?: string;
-    metricKey: string;
-    max: number;
-    period: string;
+  constructor(private http: HttpClient) {
+    this.apiUrl = environment.apiUrl;
+  }
+
+  getMetrics(): Observable<HttpResponseEnvelope<Metric>> {
+    return this.http.get<HttpResponseEnvelope<Metric>>(
+      this.apiUrl + '/Metrics'
+    );
+  }
+
+  getPeriods(): string[] {
+    return ['Hour', 'Day', 'Week', 'Month', 'Year'];
+  }
+
+  createTierQuota(
+    quota: TierQuota,
+    tierId: string
+  ): Observable<HttpResponseEnvelope<TierQuota>> {
+    return this.http.post<HttpResponseEnvelope<TierQuota>>(
+      this.apiUrl + '/Tiers/' + tierId + '/Quotas',
+      quota
+    );
+  }
 }
 
 export interface Metric {
-    id: string;
-    key: string;
-    displayName: string;
+  id: string;
+  key: string;
+  displayName: string;
+}
+
+export interface TierQuota {
+  id: string;
+  metric: Metric;
+  max: number;
+  period: string;
+}
+
+export interface Quota {
```

Comment:
What's this for?

Reply:
On the quotas-for-identities branch, there was a major overhaul of the Quotas interfaces. We separated the tier and individual quotas, in between other things. These changes have not yet been merged to main, so if I delete this class and start making the required changes, it could become a bit messier to merge.

Reply:
Okay, got it. Then leave it as it is.